wheelchair:
  z_position_upper_chassis_controller:
      type: effort_controllers/JointPositionController
      joint: upper_chassis_joint
      pid: {p: 14000.0, i: 0.0, d: 1000.0}
  
  back_controller:
      type: effort_controllers/JointPositionController
      joint: back_joint
      pid: {p: 2000.0, i: 0.0, d: 0.0}